v2/tools/generator/internal/config/multi_matcher_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package config
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestMultiMatcher_GivenDefinition_MatchesExpectedStrings(t *testing.T) {
```

Comment:
Add another test for the WasMatched behaviour? I realise that requires indirectly testing another of the matches, but otherwise I don't think there's a test checking for the non-short-circuiting behaviour - it might just disappear unless someone reads the comment.

Reply:
Added test. Good idea.